tutorial/markdown/nodejs/quickstart-typescript/quickstart-typescript.md

```diff
@@ -0,0 +1,483 @@
+---
+# frontmatter
+path: "/tutorial-quickstart-nodejs"
```

Comment:
The path is wrong. It could overwrite existing tutorial

Reply:
Updated it.